web2/app/(Creating)/create-offers/[id]/page.tsx

```diff
@@ -39,14 +38,14 @@ const useCreateOffersStyles = makeStyles(() => ({
 const CreateOffers: React.FC = () => {
   const classes = useCreateOffersStyles();
 
-  const [getNfts, { data }] = useSearchItemsLazyQuery();
+  const [getNfts, { data }] = useSearchItemsLazy();
 
   const router = useRouter();
   const { id } = useParams();
   const { offers } = useAppSelector(selectCreateOffer);
   const dispatch = useAppDispatch();
 
-  const currentOffer = offers.find((o) => o.id === Number(id));
+  const currentOffer = offers.find((o) => o.id === +id);
```

Comment:
Use explicit casting
```suggestion
  const currentOffer = offers.find((o) => o.id === Number(id));
```